CHANGELOG.rst

```diff
@@ -3,33 +3,48 @@
 0.20.0
 ------
 
-This release includes improvements to the UX on the UI, internal changes to the Knowledge Graph, a bugfix on the core service
-and support for running sessions in a separate namespace.
+Renku ``0.20.0`` tidies up the RenkuLab session start sequence, as well as an assortment of improvements and big fixes. 
 
-Features
-~~~~~~~~
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
 
-* **Renku core service**: add workflow export endpoint
-* **Renku CLI**: inform users about push in renku save command
-* **UI**: improve UX around sessions 💻 (unified start flow, better logs, error handling)
-* **UI & Gateway**: adopt the latest layout on login and logout pages 🎨
-* **Sessions**: support for running sessions in a separate namespace 
-* **Sessions**: caching jupyter servers
+**🌟 New Features**
 
-Bug fixes
-~~~~~~~~~~
+* *None this release*
+
+**✨ Improvements**
+
+* 🚀 **UI**: We tidied up the session launch sequence to have a unified appearance, regardless of where on RenkuLab you start your session from. Sessions also have better logs and error handling.
+* 🎨 **UI**: The RenkuLab login and logout pages have been updated with the latest styling.
+* 💬 **Renku CLI**: When you run ``renku save`` in a clean but unpushed repository, ``renku`` now informs you that it has pushed changes to the remote, rather than just saying that there were no changes to save.  
```

Comment:
You start with a verb in all the following bullet points, but you use different constructions for the first 3 points. (`We tidied up` vs `The ... pages have` vs ` When you run`). Would it make sense to even out by always starting with a verb?

Reply:
Not sure if that would be too limiting for the "Improvements" and "New Features" sections :thinking: